src/extensions/common/VSplitter.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { FunctionComponent, ReactElement, useState } from 'react';
```

Comment:
I added this VSplitter class but didn't actually use it (instead made a custom solution). But I preserve this code for future use. The VSplitter.tsx is just a copy of Splitter.tsx with a few minor adjustments to make a vertical splitter rather than horizontal.